src/FontManager.cpp

```diff
@@ -61,12 +61,38 @@ void FontManager::loadFonts(const QString& folder)
     QDir dir = folder;
     dir.setNameFilters(filters);
 
-    foreach (QString fontfile, dir.entryList(QDir::Files | QDir::Readable | QDir::NoDotAndDotDot)) {
-        QString fontFilePathName = QStringLiteral("%1/%2").arg(dir.absolutePath(), fontfile);
-        if (QFontDatabase::addApplicationFont(fontFilePathName) == -1) {
-            // At the point that addFonts() is called we have a GUI application
-            // in the making and can use qDebug() and not rely on iostream class
-            qWarning() << "FontManager::loadFonts() ERROR - Could not load the font in the file: " << fontFilePathName;
-        }
+    foreach (QString fontFile, dir.entryList(QDir::Files | QDir::Readable | QDir::NoDotAndDotDot)) {
+        QString fontFilePathName = QStringLiteral("%1/%2").arg(dir.absolutePath(), fontFile);
+        loadFont(fontFilePathName);
+    }
+}
+
+void FontManager::loadFont(const QString& filePath) {
```

Comment:
Have we changed our function/method opening '{' position policy - I thought we had them on the next line? 😕

Reply:
Adjusted.